src/libs/E2E/tests/chatOpeningTest.e2e.ts

```diff
@@ -29,11 +30,13 @@ const test = (config: NativeConfig) => {
         const [renderChatPromise, renderChatResolve] = getPromiseWithResolve();
         const [chatTTIPromise, chatTTIResolve] = getPromiseWithResolve();
 
-        Promise.all([renderChatPromise, chatTTIPromise]).then(() => {
-            console.debug(`[E2E] Submitting!`);
+        Promise.all([renderChatPromise, chatTTIPromise])
+            .then(waitForActiveRequestsToBeEmpty)
+            .then(() => {
+                console.debug(`[E2E] Submitting!`);
 
-            E2EClient.submitTestDone();
-        });
+                E2EClient.submitTestDone();
```

Comment:
Do you think it makes sense to move the `waitForActiveRequestsToBeEmpty` to `E2EClient.submitTestDone()`?
I think its a detail that someone who writes a test shouldn't think about / its safer if we do it by default?

Reply:
@hannojg yeah, you're absolutely right - re-worked it 👍